realm/realm-library/src/main/java/io/realm/internal/MixedNativeFunctions.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.internal;
+
+import io.realm.Mixed;
+
+
+public interface MixedNativeFunctions {
```

Comment:
It seems a bit like these are dangling here. Aren't there existing native wrappers where it fits. There is also no state in the implementation, so why not just have static methods on the OsObjectBuilder?

Reply:
The reason is because of package visibility. Mixed sits in `io.realm` and we need to get its pointer from OsObjectBuilder `io.realm.internal.objectstore`. 
This is what I came up to avoid exposing the `getNativePointer` as a public method in Mixed... do you have any suggestion??

Reply:
Ok. But still seems like there is no state in the implementation so why not just have a helper class with static-only methods in it?